angular/core/components/sidebar/sidebar.coffee

```diff
@@ -0,0 +1,6 @@
+angular.module('loomioApp').directive 'sidebar', (SidebarController)->
+  scope: false
+  restrict: 'E'
+  templateUrl: 'generated/components/sidebar/sidebar.html'
+  replace: true
+  controller: SidebarController
```

Comment:
I don't think we should need two files for this. All of the other components define the directive and its controller in the same file?